utils/treesitter/grammar.js

```diff
@@ -286,53 +294,55 @@ module.exports = grammar({
         choice(
           seq($._expression, '==', $._expression),
           seq($._expression, 'impls', $._expression),
-          seq($.designator, '=', $._expression),
+          seq($._expression, '=', $._expression),
+          // FIXME: logical and conflicts with this
```

Comment:
Is this a FIXME that will be fixed before submit, or a TODO that will be fixed later?

Reply:
```suggestion
          // TODO: Fix conflict with logical and
```